.gitignore

```diff
@@ -13,6 +13,9 @@ common.pyc
 *.komodoproject
 /nbproject/private/
 
+# Editor
+.vscode
```

Comment:
Is a `.vscode` file a common editor artifact? Or would it be better to just not commit the vscode file and remove these lines?

Reply:
Hi, vscode is a fairly popular editor, I merely added it here in the gitignore file as I contribute to a few different repositories, each with their own eslint/tslint/prettier/editorconfig setup bound to a potential editor like atom/vscode/intellij. adding the .vscode file in the pulled repository allows me to contribute to this project without automatically reformatting the js codebase according to my default editor settings. 

I don't think there's harm in ensuring that :)